requirements-dev.txt

```diff
@@ -13,7 +13,8 @@ langchain>=0.1,<0.2; python_version>="3.9"
 openai>=0.27,<=0.28.1; python_version>="3.9"
 
 # Necessary for running all local models on GPU.
-transformers[sentencepiece]>=4.0.0
+# TODO: transformers > 4.38 causes bug in model handling due to unknown factors. To be investigated.
```

Comment:
Could you make this into an issue as well?

Reply:
https://github.com/explosion/spacy-llm/issues/463